tests/posix/common/src/pthread.c

```diff
@@ -143,6 +143,13 @@ static void *thread_top_exec(void *p1)
 	return NULL;
 }
 
+static void *thread_alive_1_s(void *p1)
+{
+	sleep(ONE_SECOND);
```

Comment:
Rather than 1s, it might be feasible to sleep for 100ms.

The only reason I suggest this, is that long delays in CI tend to grow significantly when there are several testsuite configurations and permutations.

But the